src/main/java/model/Response.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Response {
+    private String protocol = "HTTP/1.1";
+    private String status;
+    private String contentType;
+    private Long contentLength = 0L;
+    private byte[] body;
+
+    public String getProtocol() {
+        return protocol;
+    }
+
+    public void setProtocol(String protocol) {
+        this.protocol = protocol;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public void setStatus(String status) {
+        this.status = status;
+    }
+
+    public String getContentType() {
+        return contentType;
+    }
+
+    public void setContentType(String contentType) {
+        this.contentType = contentType;
+    }
+
+    public int getContentLength() {
+        int getContentLength = (this.getBody() != null) ? this.getBody().length : 0;
```

Comment:
`getContentLength` 는 이 메서드의 return 값을 잠시 저장해두는 용도밖에 안되는군요. 반복되거나 동일 메서드에서 로직수행을 위한 참조가 필요하지 않다면 변수화 하지않고 조금 더 간략하게 작성하는게 좋습니다.
아래와 같은 식으로 구성해보는 건 어떤가요?
```suggestion
       return (this.getBody() != null) ? this.getBody().length : 0;
```

Reply:
https://github.com/kakao-2022-summer/be-w234-was/pull/59/commits/ebf8d097ef36b26b4b353673e88cc3162a70c312 여기에서 수정이된거죠?